### DISCUSSION TITLE
List operators do not exist

### DISCUSSION THREAD

**Discussion by nyc-de** [duckdb/duckdb#8549] [Q&A] [answered]
I feel like asking this here is overkill but Googling duckdb yields pages and pages of duckdb docs.

This is an example from the list operators documentation.

SELECT [1,2,3,4,5] && [2,5,5,6];
Error: Catalog Error: Scalar Function with name && does not exist!
Did you mean "&"?
LINE 1: select [1,2,3,4,5] && [2,5,5,6];

It doesn't copy over but the error message also includes the hat symbol pointing to the first & in the operator.

It's me right?

Any help would be appreciated.

**Comment by Tishj** [comment, marked as answer]
You're looking at the "dev" version of the docs, that's not in 0.8.1
